variables.tf

```diff
@@ -26,6 +26,12 @@ variable "azureopenai_customsubdomain_name" {
   description = "The subdomain name used for token-based authentication. Changing this forces a new resource to be created. Leave this variable as default would use a default name with random suffix."
 }
 
+variable "access_from_private_endpoint" {
+  type        = bool
+  default     = false
+  description = "Specifies whether access to the Azure OpenAI account is done throught private endpoint. Changing this forces a new resource to be created. Default to false."
```

Comment:
"Changing this forces a new resource to be created". Should it be more specific? From the patch it seems to impact only the `azurerm_private_dns_zone` resource.

Reply:
Can you explain why you need this bool variable ? Why is not possible to check if `var.private_endpoint` value is `{}` ?

Like this you risk that the user configures `var.private_endpoint = {}` (default value) and `access_from_private_endpoint = True` and this should be an impossible configuration.